src/lib/netlist/macro/nlm_roms_lib.cpp

```diff
@@ -342,6 +342,40 @@ static NETLIST_START(PROM_MK28000_DIP)
 	ALIAS(24, A.OE1)
 NETLIST_END()
 
+//- Identifier:  ROM_MCM14524_DIP
+//- Title: MCM14524 1024-BIT READ ONLY MEMORY
+//- Pinalias: CLK,EN,B0,B1,B2,B3,A2,GND,A3,A4,A5,A6,A7,A1,A0,VCC
+//- Package: DIP
+//- Param: ROM
+//-    The name of the source to load the rom content from
+//- NamingConvention: Naming conventions follow Motorola datasheet
+//- Limitations:
+//-    None.
```

Comment:
When supply voltage depending timing is not implemented this should be stated as a limitation.

Reply:
Done, all the new models are marked if they do not support voltage dependent timing